.github/ISSUE_TEMPLATE/bug-report-.md

```diff
@@ -0,0 +1,47 @@
+---
```

Comment:
Would you mind renaming the file to `bug_report.md` just for consistency with other repos?

Reply:
Done!